Makefile

```diff
@@ -47,27 +47,23 @@ restart-detached:
 	make down
 	make up-detached
 
-devstack-install: validate-devstack-folders ## install local version of package into docker devstack for development
```

Comment:
Confirming this means that studio-frontend no longer needs to be nested in pre-existing folder structure, right?

Reply:
yes. With tyler's incoming changes, you can have sfe anywhere relative to devstack and they'll be able to talk